pkg/server/config/config.go

```diff
@@ -107,7 +107,7 @@ func NewConfig() *Config {
 func (s *Config) Validate() []error {
 	var errs []error
 
-	if s.Datastore.Type != "mongodb" && s.Datastore.Type != "kubeapi" {
```

Comment:
IMO we can add to-be-implemented methods like in mysql.go. But we should not allow user to use it before implementing. We can postpone it until its implementation is added.

Reply:
Yeah, I will change them to to-be-implemented in mysql.go.
And  for now will only add commits to the `/mysql` only. And after implementation of all the methods we can allow users to use.

Since this was a draft PR and its not getting merged until all methods are implemented , I thought this will not effect the users.

Reply:
I see

Reply:
Added to-be-implemented errors in mysql.go